modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/api/ItSqlAsynchronousApiTest.java

```diff
@@ -173,16 +188,79 @@ public void ddl() throws ExecutionException, InterruptedException {
     public void dml() throws ExecutionException, InterruptedException {
         sql("CREATE TABLE TEST(ID INT PRIMARY KEY, VAL0 INT)");
 
-        IgniteSql sql = CLUSTER_NODES.get(0).sql();
+        IgniteSql sql = igniteSql();
         Session ses = sql.createSession();
 
+        SqlQueryProcessor qryProc = (SqlQueryProcessor) IgniteTestUtils.getFieldValue(sql, IgniteSqlImpl.class, "qryProc");
```

Comment:
it's a bit mess, let's introduce separate tests

Reply:
done